weave/trace/weave_client.py

```diff
@@ -534,6 +552,17 @@ class ApplyScorerSuccess:
             wc._send_score_call(self, score_call, scorer_ref_uri)
         return apply_scorer_result
 
+    def to_dict(self) -> CallDict:
+        d = {}
+        for field in dataclasses.fields(self):
+            if field.name.startswith("_"):
+                continue
+            d[field.name] = getattr(self, field.name)
+        d["op_name"] = self.op_name
+        d["display_name"] = self.display_name
+
+        return cast(CallDict, d)
```

Comment:
this cast is not guaranteed. I would do something to at least validate the keys

Reply:
What case do you see missing here?